tests/functional/test_redemption.py

```diff
@@ -136,13 +136,13 @@ def test_sweep(d_yfi, yfi, redemption, gov):
 def test_oracle(project, yfi, d_yfi, ve_yfi, gov):
     mock = project.MockOracle.deploy(sender=gov)
     redemption = project.Redemption.deploy(
-        yfi, d_yfi, ve_yfi, gov, mock, mock, 10 * AMOUNT, sender=gov
+        yfi, d_yfi, ve_yfi, gov, mock, 10 * AMOUNT, sender=gov
```

Comment:
since unit tests mostly using the mock oracle and we had the contract live we have some sample data of actual values return

is it to much work to add to ape a fork test using this same account and value and check that return with new contract matches reedem expectations?

since test can be integration only on specific block number, to check how this contract new only CL feed would have behaved against these past trxs

https://etherscan.io/tx/0x51613ef0c083e8c133a2d71fa628db4f4c80c8462e2b5efce05f0f3b0faa9d78

Reply:
I have added a test [here](https://github.com/yearn/veYFI/pull/232/files#diff-c53faeec52cfa9641e6ab842d93d6ccd65524569f39252416f452e071915b124R175) that compares the oracle price in the old and new redemption contract, after making sure the old one uses the chainlink price. This should be an equivalent test without having to run a test on an old block, which is (afaik) not very easy to do